unity-client/Assets/ABConverter/Tests/ABConverterCoreShould.cs

```diff
@@ -0,0 +1,370 @@
+using DCL;
+using NUnit.Framework;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Text.RegularExpressions;
+using UnityEngine;
+using UnityEngine.Events;
+using UnityEngine.TestTools;
+using UnityEngine.UI;
+using UnityGLTF.Cache;
+
+namespace ABConverterTests
+{
+    public class ABConverterShould
+    {
+        [UnityTest]
+        public IEnumerator DumpAreaCorrectly()
+        {
+            //TODO(Brian): Implement later 
+            yield break;
+        }
+
+        [UnityTest]
+        public IEnumerator DumpSceneCorrectly()
+        {
+            //TODO(Brian): Implement later 
+            yield break;
+        }
+    }
+
+    public class ABConverterCoreShould
+    {
+        private ABConverter.Core core;
+        private ABConverter.Environment env;
+
+        string basePath = @"C:\test-path\";
```

Comment:
Is this absolute path okay?

Reply:
yep, its just used for mocking, we aren't creating any real files in the test.